manifest.yaml

```diff
@@ -42,7 +42,13 @@ arguments:
     schema:
       type: array
       items:
-        type: string
+        anyOf:
```

Comment:
[stencil-outreach](https://github.com/getoutreach/stencil-outreach/blob/d8af105a02820d09079229b67a472ddfa71162c2/manifest.yaml#L41-L46) needs to be updated as well